### PR TITLE
feat(sponsors/models.py): Add a new field of sponsor subtitle

### PR DESCRIPTION
**Pull request by hsuanchi**
## Types of changes

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
- Add a new field for sponsor subtitle (e.g. `連續N年贊助`)
  - support zh/en
  - the data needs to be accessed via [API](https://github.com/pycontw/pycon.tw/blob/master/src/sponsors/views.py#L21)
  - ref: [the field of sponsor intro](https://github.com/pycontw/pycon.tw/blob/master/src/sponsors/models.py#L26_L28)

https://trello.com/c/vICfFVL6/31-add-a-new-field-of-sponsor-subtitle

## Steps to Test This Pull Request
1. Go to Django Admin.
2. Create Sponsor.

### CODE REVIEW

**Review by kaka-lin — COMMENTED** [review]
LGTM!

Remember rebase to new master

**Comment by hsuanchi** [comment]
> LGTM!
> 
> Remember rebase to new master

Done, Thanks!

File: src/sponsors/translation.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 
 class SponsorTranslationOptions(TranslationOptions):
-    fields = ('intro',)
+    fields = ('intro','subtitle',)
```
Comment (setmao) [inline src/sponsors/translation.py:7]: Typo, missing whitespace after ‘,'

**Review by setmao — CHANGES_REQUESTED** [review]

File: src/sponsors/translation.py (hunk 1 above)
Comment (hsuanchi) [reply src/sponsors/translation.py]: Done, Thanks!

**Review by setmao — APPROVED** [review]
LGTM！

**Pull request merged**